### ISSUE TITLE
Note in Accessible Authentication needs to be reworded

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#2714]
The note below the normative text for [3.3.7 Accessible Authentication](https://www.w3.org/TR/WCAG22/#accessible-authentication) reads:
> Objects to recognize and user provided content may be represented by images, video, or audio.

The sentence makes no sense; I can't figure out exactly what it is _supposed_ to mean. Can it be rephrased to be easily understood?

**Comment by bruce-usab** [comment]
File type is not the point of the note, but maybe it should be two sentences?

**Comment by bruce-usab** [comment]
Object Recognition:  The cognitive function test is to recognize objects.

Note:  Objects to be recognized may be represented by images, video, or audio.

**Comment by bruce-usab** [comment]
> Personal Content:  The cognitive function test is to identify non-text content the user provided to the website.

Note: Content the user provides to the website may be images, video, or audio.

**Comment by mbgower** [comment]
How about:
> Objects and personal content to be recognized may be represented by images, video, or audio.

**Comment by mbgower** [comment]
Or maybe this problem points to a normative wording change, such as...

**Recognition**
The cognitive function test is to recognize objects or identify non-text content the user provided to the website.

**Comment by bruce-usab** [comment]
Putting Image/video/audio at beginning of note might be clearer:

> Note:  Images, video, and audio are examples of content the user might provide to a website or be used for object recognition.

**Comment by bruce-usab** [comment]
I like combining Object Recognition and Personal Content.

**Comment by mbgower** [comment]
I will create a PR for this so it can be surveyed.

**Comment by awkawk** [comment]
It would probably help if you can find a comment that was entered during the WCAG 2.2 CR review. If not, unless this is editorial we may not be able to change it without returning to pre-CR

**Comment by mbgower** [comment]
@awkawk i would consider this an editorial candidate. There is no change in meaning. All that has happened is that the wording of 2 bullets has been carried over verbatim to a single, combined bullet. The only addition/change is the descriptor at the front of the bullet ("Easy recognition" instead of "Object recognition" and "Personal content")

**Comment by bruce-usab** [comment]
@awkawk, since you have some experience with this, do you think we can change the note without risking returning to pre-CR?

**Comment by awkawk** [comment]
I don't like the proposed change to combine bullets 3 and 4. I'm worried that this will also require a change to the note, and that seems like more of a rewrite.

The 3rd and 4th bullets are clear enough so I think that Mike was on the right track. How does this sound?

Tests involving personal content or object recognition may use images, video, or audio.

**Comment by mbgower** [comment]
Not going to lose any sleep if we go that way, @awkawk, but it does infer that an audio sample can be used for object recognition, which I don't _think_ was the intent?

**Comment by yatil** [comment]
With all the open issues and the apparent disengagement by the working group fixing these issues, I think there is a need for rework and creating a new Candidate Recommendation Snapshot that addresses the numerous issues. Sorry for being a downer on this, but the whole update feels half-baked.

**Comment by alastc** [comment]
We're likely to re-run CR, but even so I'd prefer to keep this to updating the note editorially. I think combining the bullets would decrease readability.
